.changeset/nasty-tables-visit.md

```diff
@@ -0,0 +1,36 @@
+---
+'rocket-preset-extend-lion-docs': patch
```

Comment:
this package does not export a custom element manifest as it's not a custom element (the found cases are test fixtures)

```suggestion

```

Reply:
ah, so no bump needed indeed

Reply:
Ai, didn't see that